config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml

```diff
@@ -59,6 +59,10 @@ presets:
   # Dump full systemd journal on master and nodes.
   - name: LOG_DUMP_SYSTEMD_JOURNAL
     value: "true"
+  # Maximal time the log dumping over SSH operation can take.
+  # Unresponsive API server caused an extremely long iteration over nodes and a test timeout.
```

Comment:
I like your comment as it gives more relevant context than the one I proposed. There are situations in which the creation of logexporter daemonset succeeds but the fallback to log dumping over SSH still takes place, though. WDYT about that one:
```
Timeout for the log dumping over SSH. Relevant only if fallback to SSH log dumping takes place 
e.g. when logexporter daemonset fails for some reason.
We deliberately cap it at 1h to avoid spending too much time (e.g. over 5h for 5k node cluster)
on dumping logs that in most cases we won't need anyway. 
```

Reply:
It's great, thanks!